tracing/src/main/java/org/axonframework/extensions/tracing/OpenTraceDispatchInterceptor.java

```diff
@@ -9,6 +39,11 @@
 import java.util.Optional;
 import java.util.function.BiFunction;
 
+/**
+ * A {@link MessageDispatchInterceptor} which maps the {@link SpanContext} to {@link org.axonframework.messaging.MetaData}
+ *
+ * @author Christophe Bouhier
```

Comment:
Nit: I think it's nice to have a `@since` too.

Reply:
will do, thx